cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service

```diff
@@ -1,8 +1,8 @@
 [Unit]
 Description=kubelet: The Kubernetes Node Agent
 Documentation=https://kubernetes.io/docs/home/
-Wants=network-online.target
-After=network-online.target
+Wants=network-online.target etcd.service docker.service containerd.service crio.service
+After=network-online.target etcd.service docker.service containerd.service crio.service
```

Comment:
If we hope to ensure kubelet start as late as possible we could say (because CRI usually come right after `network-online.target`):
```
Wants=multi-user.target
After=multi-user.target
```

Anyway, since these 3 CRI are Kubernetes recent supported, listing them all here should be valid (yes, but not elegant enough...)

P.S. `etcd.service` should also be listed here, since we are now supporting both Internal/External Etcd cluster.

Reply:
I try to postpone kubelet start as "After=network-online.target remote-fs.target", which should be good enough.